src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -31,7 +31,8 @@ public void update(MatchPlayer player) {
     }
   }, // Changes if observers are visible
   SOUNDS("sounds", SOUNDS_ON, SOUNDS_OFF), // Changes if sounds are played
-  ;
+  VOTE_BOOK("vote-book", VOTES_ON, VOTES_OFF) // Changes if the vote book is shown on cycle
```

Comment:
`VOTE("vote")` -- similar to `PICKER`, when we have a GUI for settings we'll explain it.

Reply:
I decided to use `vote_book` since it was a bit more descriptive than just `vote`. Like toggling the "picker" is a bit more obvious than toggling "vote". 

I can still change it if you wish

Reply:
I've been trying to keep setting keys a single word :)